docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -51,24 +51,18 @@ Describe "One is really one" {
 }
 ```
 
-If you are checking for proper errors, do that in a `try/catch`, and then check `FullyQualifiedErrorId`. Checking against `FullyQualifiedErrorId` is recommended because it does not change based on culture as an error message might.
+If you are checking for proper errors, use the `ShouldBeErrorId` helper defined in HelpersCommon.psm1 module which is in your path if you import `build.psm1`.
+Checking against `FullyQualifiedErrorId` is recommended because it does not change based on culture as an error message might.
 
 ```powershell
 ...
-it "Get-Item on a nonexisting file should have error PathNotFound" {
```

Comment:
Sometimes there are scenarios where we can't use `ShouldBeErrorId`. I think we need to leave this as an alternative.